searx/templates/simple/preferences/answerers.html

```diff
@@ -33,10 +33,10 @@
     {%- for plugin in plugins -%}
       {%- if plugin.preference_section == 'query' -%}
         <tr>{{- '' -}}
-          <td class="checkbox-col">{{- checkbox_onoff_reversed('plugin_' + plugin.id, plugin.id not in allowed_plugins) -}}</td>{{- '' -}}
+          <td class="checkbox-col">{{- checkbox_onoff_reversed('plugin_' + plugin.id, plugin.id not in allowed_plugins, 'plugin_labelledby' + plugin.id) -}}</td>{{- '' -}}
           <td>{{ plugin.query_keywords|join(', ') }}</td>{{- '' -}}
           <td>{{ _(plugin.name) }}</td>{{- '' -}}
-          <td>{{ _(plugin.description) }}</td>{{- '' -}}
+          <td id="{{ 'plugin_labelledby' + plugin.id }}">{{ _(plugin.description) }}</td>{{- '' -}}
```

Comment:
We have the same ID used in the preferences template. Is there any risk in ID collision in answers and preferences?

Reply:
The `id` is build up from the prefix `plugin_labelledby` plus the value from the `plugin.id` .. since IDs of plugins are unique, I don't expect a ID collision from those of *answers* and *preferences*.